components/gsma-authenticators/src/main/java/com/wso2telco/gsma/authenticators/voice/VoiceIVRFutureCallback.java

```diff
@@ -0,0 +1,88 @@
+package com.wso2telco.gsma.authenticators.voice;
+
+import com.wso2telco.gsma.authenticators.AuthenticatorException;
+import com.wso2telco.gsma.authenticators.Constants;
+import com.wso2telco.gsma.authenticators.DBUtils;
+import com.wso2telco.gsma.authenticators.util.BasicFutureCallback;
+import com.wso2telco.ids.datapublisher.model.UserStatus;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.wso2telco.ids.datapublisher.model.UserStatus'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.wso2telco.com:9000/coding_rules#rule_key=squid%3AUselessImportCheck)